doc/modules/cassandra/pages/architecture/snitch.adoc

```diff
@@ -18,9 +18,9 @@ following properties on `cassandra.yaml`:
 
 * `dynamic_snitch`: whether the dynamic snitch should be enabled or
 disabled.
-* `dynamic_snitch_update_interval_in_ms`: controls how often to perform
+* `dynamic_snitch_update_interval`: controls how often to perform
 the more expensive part of host score calculation.
-* `dynamic_snitch_reset_interval_in_ms`: if set greater than zero, this
+* `dynamic_snitch_reset_interval`: if set greater than zero, this
```

Comment:
Especially now that we've removed the units from this, would it be helpful to list the defaults? (I guess that would be `100ms` and `10m` in this case?)